library_generation/test/integration_tests.py

```diff
@@ -176,6 +187,11 @@ def __build_image(cls, docker_file: str, cwd: str):
             cwd=cwd,
         )
 
+    @classmethod
+    def __remove_generated_files(cls):
+        shutil.rmtree(f"{output_dir}", ignore_errors=True)
```

Comment:
I tried this in the past and it did not work due to permissions. Can you remove `ignore_errors=True` and see if it works? Separately, why did we add `ignore_errors=True` in the first place? I think we should error out instead of ignore errors in integration tests.

Reply:
Right now the permission issue is still there and will be resolved in #2615.

If the remove failed, the integration test will fail when running `git clone` in an non-empty directory.

Reply:
So does it work or not at this moment? If we still have permission issues, I don't think we should add this line right now.

Reply:
I comment out this line.